components/search_hint/search_hint.tsx

```diff
@@ -64,6 +68,8 @@ const SearchHint = (props: Props): JSX.Element => {
                             <button
                                 className={classNames({highlighted: props.highlightedIndex === 1})}
                                 onClick={() => props.onSearchTypeSelected && props.onSearchTypeSelected('files')}
+                                onBlur={() => props.onElementBlur && props.onElementBlur()}
+                                onFocus={() => props.onElementFocus && props.onElementFocus()}
```

Comment:
Nit, for these calls, we can just pass the handler directly. e.g. instead of 

```ts
onFocus={() => props.onElementFocus && props.onElementFocus()}
```

```ts
onFocus={props.onElementFocus}
```

The code is a bit shorter, it saves a function allocation, and if the prop function is undefined, no handler ends up being created/attached to the element.